homework/hw2-goals-approaches/ez/multilabel.py

```diff
@@ -0,0 +1,41 @@
+"""Codes for multilabel model"""
+
+# packages
+import pandas as pd
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn import preprocessing
+from sklearn.model_selection import train_test_split
+from sklearn.svm import LinearSVC
+import pickle
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.svm import SVC
+from sklearn.multiclass import OneVsRestClassifier
+
+# Load the data
+df = pd.read_feather("../../../data/mtg.feather")
+
+# X
+tfidf = TfidfVectorizer(
+    min_df=5, # ignore rare words (appear in less than 5 documents)
+    max_df=0.8, # ignore common words (appear in more than 80% of documents)
+    stop_words="english"
+)
+text = df["text"] + df["flavor_text"].fillna('')
+X = tfidf.fit_transform(text)
+
+# y
+ci = df["color_identity"]
+cv = CountVectorizer(tokenizer=lambda x: x, lowercase=False)
+y = cv.fit_transform(ci)
```

Comment:
Generally I see [`MultiLabelBinarizer`](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.MultiLabelBinarizer.html) used for this, any advantage/reason for a coutvec here instead? Generally we do not allow counts more than 1 in multilabel predictions.